dashboard/scripts/import_workshop_organizer_regional_partners.rb

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+require_relative '../config/environment'
+require 'csv'
+
+COL_NAME = 'PLP Name'
```

Comment:
I wonder if we should rename this column now to `Partner Name`? Not necessary in this PR.